sklearn/model_selection/_split.py

```diff
@@ -1597,3 +1597,21 @@ def _build_repr(self):
         params[key] = value
 
     return '%s(%s)' % (class_name, _pprint(params, offset=len(class_name)))
+
+
+ALL_CVS = {'KFold': KFold,
```

Comment:
The idea was to make it public like the constants in metrics I think... Do you feel it would not add much value being public?